goosebit/settings.py

```diff
@@ -2,23 +2,24 @@
 from dataclasses import dataclass
 from pathlib import Path
 
-import yaml
 from argon2 import PasswordHasher
+from dynaconf import Dynaconf
 from joserfc.jwk import OctKey
 
 from goosebit.permissions import Permissions
 
 BASE_DIR = Path(__file__).resolve().parent.parent
-TOKEN_SWU_DIR = BASE_DIR.joinpath("swugen")
-SWUPDATE_FILES_DIR = BASE_DIR.joinpath("swupdate")
-UPDATES_DIR = BASE_DIR.joinpath("updates")
 DB_MIGRATIONS_LOC = BASE_DIR.joinpath("migrations")
 
 SECRET = OctKey.import_key(secrets.token_hex(16))
 PWD_CXT = PasswordHasher()
 
-with open(BASE_DIR.joinpath("settings.yaml"), "r") as f:
-    config = yaml.safe_load(f.read())
+config = Dynaconf(
+    envvar_prefix="GOOSEBIT",
+    settings_files=[BASE_DIR.joinpath("settings.yaml"), BASE_DIR.joinpath(".secrets.yaml")],
+)
+
+UPDATES_DIR = Path(config.get("artifacts_dir", BASE_DIR.joinpath("updates")))
```

Comment:
Might as well rename this from `UPDATES_DIR` to `ARTIFACTS_DIR`.

Reply:
I would then also rename the `updates` directory to `artifacts`. I like the name better, as the files don't necessarily have to be updates. Is that an option for you?

Reply:
> I would then also rename the `updates` directory to `artifacts`. I like the name better, as the files don't necessarily have to be updates. Is that an option for you?

:100: this is much better.